src/tests/Common/XUnitLogChecker/XUnitLogChecker.cs

```diff
@@ -120,6 +148,33 @@ static int Main(string[] args)
         return SUCCESS;
     }
 
+    static IEnumerable<string> TryReadFile(string filePath)
+    {
+        IEnumerable<string>? fileContents = null;
+        Stopwatch fileReadStopwatch = Stopwatch.StartNew();
+
+        while (fileReadStopwatch.Elapsed.Seconds < 60)
```

Comment:
Nit - while mostly functional in practice, technically this is incorrect as the <code>Elapsed</code> property returns a <code>TimeSpan</code> representing the elapsed time and only checking its <code>Seconds</code> field would for instance yield zero at a one minute mark, I believe this is the purpose of the <code>ElapsedTicks</code> and <code>ElapsedMilliseconds</code> properties on <code>Stopwatch</code>.

Reply:
Hmm, in fact, upon second reading I think it's completely incorrect as the <code>Seconds</code> property is always in the range 0-59 so this is basically an infinite loop in case the file remains locked forever.

Reply:
Fixed.